inspire/modules/formatter/views.py

```diff
@@ -36,3 +38,12 @@ def get_bibtex(recid):
     record = get_record(recid)
     bibtex = Bibtex(record).format()
     return jsonify({"result": bibtex, "recid": recid})
+
+
+@blueprint.route('/latex', methods=['GET', ])
+@wash_arguments({'recid': (int, 0),
+                 'latex_format': (text_type, "")})
+def get_latex(recid, latex_format):
+    record = get_record(recid)
```

Comment:
As I tested it in the shell it doesn't crash, just returning nothing.

2015-08-26 9:40 GMT+02:00 Javier Martin Montull <notifications@github.com>:

> In inspire/modules/formatter/views.py
> <https://github.com/inspirehep/inspire-next/pull/320#discussion_r37953703>
> :
>
> > @@ -36,3 +38,12 @@ def get_bibtex(recid):
> >      record = get_record(recid)
> >      bibtex = Bibtex(record).format()
> >      return jsonify({"result": bibtex, "recid": recid})
> > +
> > +
> > +@blueprint.route('/latex', methods=['GET', ])
> > +@wash_arguments({'recid': (int, 0),
> > +                 'latex_format': (text_type, "")})
> > +def get_latex(recid, latex_format):
> > +    record = get_record(recid)
>
> if no recid is passed and it defaults to 0, does it crash here the call
> to get_record?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/inspirehep/inspire-next/pull/320/files#r37953703>.
>